Dockerfile

```diff
@@ -42,7 +42,7 @@ RUN apt-get install -y \
 ENV JAVA_HOME /usr/lib/jvm/java-8-oracle
 
 #
-# Install python 2.7 and dependancies
+# Install python 2.7 and dependancies for Genetorrent.
 #
 
 RUN apt-get install -y python
```

Comment:
Now you should start to add your python scripts to the container. You can do this by user `ADD <resources>` from the working dir. You can verify this is working by using Docker machine, doing a `git clone` and then building the container.